app/views/casework/crime_applications/_client_details.html.erb

```diff
@@ -44,38 +44,40 @@
         <%= nino.presence || t(:not_provided, scope: 'values') %>
       </dd>
     </div>
-    <div class="govuk-summary-list__row">
-      <dt class="govuk-summary-list__key">
-        <%= label_text(:telephone_number) %>
-      </dt>
-      <dd class="govuk-summary-list__value">
-        <%= applicant_tel.presence || t(:not_provided, scope: 'values') %>
-      </dd>
-    </div>
-    <div class="govuk-summary-list__row">
-      <dt class="govuk-summary-list__key">
-        <%= label_text(:home_address) %>
-      </dt>
-      <dd class="govuk-summary-list__value">
-        <% if applicant.home_address %>
-          <%= render 'address', address: applicant.home_address %>
-        <% else %>
-          <%= t(:no_home_address, scope: 'values') %>
-        <% end %>
-      </dd>
-    </div>
-    <div class="govuk-summary-list__row">
-      <dt class="govuk-summary-list__key">
-        <%= label_text(:correspondence_address) %>
-      </dt>
-      <dd class="govuk-summary-list__value">
-        <% if applicant.correspondence_address_type == 'other_address' %>
-          <%= render 'address', address: applicant.correspondence_address %>
-        <% else %>
-          <%= t(applicant.correspondence_address_type, scope: 'values') %>
-        <% end %>
-      </dd>
-    </div>
+    <% unless crime_application.case_details.appeal_reference_number %>
```

Comment:
What if a usn is provided rather than a reference number?

Reply:
appeal_reference_number field is the type, so either "appeal_usn" or "appeal_maat_id" then there are fields with those names with the actual ids in.

Reply:
Ahhh yes 😄 thanks